src/unittests/f32_interval.spec.ts

```diff
@@ -2607,3 +2608,45 @@ g.test('dotInterval')
       `dotInterval([${x}], [${y}]) returned ${got}. Expected ${expected}`
     );
   });
+
+interface VectorToVectorCase {
+  input: number[];
+  expected: IntervalBounds[];
+}
+
+g.test('normalizeInterval')
+  .paramsSubcasesOnly<VectorToVectorCase>(
+    // prettier-ignore
+    [
+      // vec2
+      {input: [1.0, 0.0], expected: [[hexToF64(0x3feffffe, 0x70000000), hexToF64(0x3ff00000, 0xb0000000)], [hexToF32(0x81200000), hexToF32(0x01200000)]] },  // [ ~1.0, ~0.0]
```

Comment:
Bit confused by the `~0.0` here. Isn't `0 / y` always `0` for any finite `y`?

The WGSL spec says for `x / y` and f32:

> 2.5 ULP for `|y|` in the range <code>[2<sup>-126</sup>, 2<sup>126</sup>]</code>

So if `x` is zero, we get zero, no?

Reply:
Unfortunately no :-(

y = length of the vector is in range, so it should have 2.5 ULP error

so 0/y does equal 0, but the acceptance interval is [0 - 2.5 * ULP(0), 0 + 2.5 * ULP(0)]
ULP(0) is larger then you would expect, because a platform may FTZ, so the nearest value is max negative or min positive, not the nearest subnormal.

Reply:
Ack. Thanks :)